pages/candy-machines/index.tsx

```diff
@@ -34,6 +37,18 @@ const ManageCandyMachines: NextPage = () => {
         }
     }
 
+    const refreshCandyMachines = async () => {
+        setIsRefreshLoading(true)
+        if (!connection) return
```

Comment:
if there is no connection we should reset also `isRefreshLoading`, otherwise it will stay loading forever